src/modules/tools/zprobe/CartGridStrategy.cpp

```diff
@@ -196,17 +212,29 @@ bool CartGridStrategy::load_grid(StreamOutput *stream)
         return false;
     }
 
-    uint8_t size;
+    uint8_t load_grid_x_size, load_grid_y_size;
     float x, y;
 
-    if(fread(&size, sizeof(uint8_t), 1, fp) != 1) {
+    if(fread(&load_grid_x_size, sizeof(uint8_t), 1, fp) != 1) {
```

Comment:
This breaks compatibility with exisiting saved grid files. Not quite sure how you would fix this.